runtime/src/lib.rs

```diff
@@ -307,8 +278,10 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u128 = 500;
+    pub const ExistentialDeposit: u128 = 1 * MILLIUNIT;
     pub const MaxLocks: u32 = 50;
+    pub const MaxReserves: u32 = 50;
+    pub const TransactionByteFee: u128 = 1 * MICROUNIT;
 }
 
 impl pallet_balances::Config for Runtime {
```

Comment:
why isn't `AccountStore`'s value the same as standalone `type AccountStore = frame_system::Module<Runtime>;` https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR537

Reply:
I don't know. @festelo Can you answer this? I haven't changed this. I think it was according to the template.

Reply:
added to #227